content/user/apps/Session-metadata.md

```diff
@@ -0,0 +1,68 @@
+
+## Agent Configuration 
+
+To send session metadata for your application to the Contrast UI, you must add the configuration settings to your agent configuration file. The build properties that the agent can report are **branch name**, **build number**, **commit hash**, **committer**, **Git tag**, **repository**, **test run** and **version**. You may include all or some of these properties, as desired. 
+
+You can supply these settings as system properties, environment settings or properties in a YAML configuration file. See the following configuration examples for each application language.
+
+### Examples
+ 
+If you use system properties for [Java](installation-javaconfig.html), include an additional entry in the line where you add your `javaagent` flag. In this case, you will set the property `contrast.application.session_metadata` to a set of key-value pairs that identify your test run. 
+
+```
+-Dcontrast.application.session_metadata="branchName=feature/some-new-thing,committer=Jane,repository=Contrast-Java"
+```
+ 
+As a [.NET](installation-netconfig.html) developer using *app.config* or *web.config*, you can add an entry to your configuration to specify this property. 
+
+```
+<?xml version="1.0"?>
+<configuration>
+  <connectionStrings />
+  <appSettings>
+    <add key="contrast.application.session_metadata" value="branchName=feature/some-new-thing,committer=Jane,repository=Contrast-DotNet" />
+```
+ 
+If you use YAML configuration, like most [Node](installation-nodeconfig.html) and [Ruby](installation-rubyconfig.html) agents, you can add an additional entry to your *contrast_security.yaml* file. 
+
+```
+application:
+ session_metadata:branchName=feature/some-new-thing,committer=Jane,repository=Contrast-Ruby
+```
+ 
+### Continuous integration 
+
+If you don't want to set these fields locally, you can also edit your continuous integration (CI) build scripts to set these for you. In Jenkins, you can pull the build number using the `BUILD_NUMBER` property. 
+
+```
+-Dcontrast.application.session_metadata="branchName=feature/some-new-thing,committer=Jane,repository=Contrast-Java,buildNumber=$BUILD_NUMBER"
+```
+
+If you use plugins in your CI, like the [Contrast Jenkins Plugin](tools-ci.html#jenkins), you can use these to make even more dynamic updates. 
+
+```
+-Dcontrast.application.session_metadata="branchName=$GIT_BRANCH,committer=$GIT_COMMITTER_NAME,commitHash=$GIT_COMMIT_HASH,repository=$GIT_URL,buildNumber=$BUILD_NUMBER"
+```
+
+## View Data by Application
+
+To see the session metadata reported by the agent, go to your application's **Vulnerabilities** tab. The data for each vulnerability is displayed in the grid and the timeline. 
+
+Use the **View By** menu above the timeline to filter the data by the properties that you included in your agent configuration. This updates the values shown in the **Seen By** column in the grid. Use the filter for the grid column to refine the results. 
```

Comment:
It may be worth noting, that there will be no "seen by" column and no "view by" filter menu if they have no session metadata. Additionally, the "seen by" column also disappears when "dissociated" is selected in the "view by" filter when there is metadata, i.e., if there is no metadata to show the seen by column disappears.